cms/jinja2/templates/components/streamfield/image_block.html

```diff
@@ -1,4 +1,57 @@
-{% image(value.image, "width-1000", 'loading="lazy"' %}
-{% if value.caption %}
-    <p>{{ value.caption }}</p>
-{% endif %}
+<!--
+    What is considered a large vs small image can vary based on the context of use.
+-->
+{# templates/components/streamfield/image_block.html #}
+{% set large_img = image(value.image, "width-800") %}
+{% set small_img = image(value.image, "width-400") %}
+
+{% from "components/image/_macro.njk" import onsImage %}
+
+{# fmt:off #}
+<div>
+    {# Figure Title #}
+    {% if value.figure_title %}
+        <h3 class="ons-chart__title">{{ value.figure_title }}</h3>
+    {% endif %}
+
+    {# Figure Subtitle #}
+    {% if value.figure_subtitle %}
+        <p>{{ value.figure_subtitle }}</p>
+    {% endif %}
+
+    {# Image #}
+    {{
+        onsImage({
+            "image": {
+                "smallSrc": small_img.url,
+                "largeSrc": large_img.url
+            },
+            "alt": value.image.description,})
```

Comment:
NP: Formatting
```suggestion
        onsImage({
            "image": {
                "smallSrc": small_img.url,
                "largeSrc": large_img.url
            },
            "alt": value.image.description
        })
```

PS. I think we are a bit inconsistent about the trailing comma, but it looks we mostly avoid it, so I removed it in my suggestion.